examples/gitserver/hooks/post-receive

```diff
@@ -7,48 +7,77 @@ set -o pipefail
 function key {
   git config --local --get "${1}"
 }
+
 function addkey {
   git config --local --add "${1}" "${2}"
 }
 
 function detect {
-  if detected=$(key openshift.io.detect); then
-    exit 0
-  fi
-  if ! url=$(key gitserver.self.url); then
-    echo "detect: no self url set"
-    exit 0
+  local repoURL="$1"
+  local repoName="$2"
+  local detectionScript=${DETECTION_SCRIPT:-}
```

Comment:
If $DETECTION_SCRIPT is not set and I try to reference it, I believe my script will break